outputs.tf

```diff
@@ -30,6 +30,7 @@ output "seed_project_id" {
 output "terraform_sa_email" {
   description = "Email for privileged service account for Terraform."
   value       = google_service_account.org_terraform.email
+  depends_on  = [module.seed_project.enabled_apis]
```

Comment:
@apeabody I maybe misunderstanding, but didnt we want to wait for IAM resources rather than service enablement per https://github.com/terraform-google-modules/terraform-google-bootstrap/issues/160?

Reply:
Yes - I just misremembered :)